taxcalc/tests/test_records.py

```diff
@@ -18,6 +18,46 @@ def test_create_records():
     r = Records(tax_dta_path)
     assert r
 
+
 def test_create_records_from_file():
     r = Records.from_file(tax_dta_path)
     assert r
+
+
+def test_imputation():
```

Comment:
I think a small vector test would be preferred to a scalar, but I don't think that should hold up the commit.